src/consts.h

```diff
@@ -35,6 +35,7 @@
 #define IMU_BMI160 8
 #define IMU_ICM20948 9
 #define IMU_ICM42688 10
+#define IMU_LSM6DSV 12
```

Comment:
This should be noted when merging, depending on the order of merging PRs this might be awkward to skip a number.